contrib/collectd/patches/remove_netlink_constant.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/netlink.c	2020-09-03 11:09:35.172575031 +0000
```

Comment:
what does this patch fix?

Reply:
IIRC (will find the issue on the collectd repo) this constant isn't in the standard so musl doesn't provide it, and it breaks the build on clang.

Reply:
right, that sounds correct :)

looks ok